templates/events/mail_participants.html

```diff
@@ -14,37 +14,15 @@ <h2>Send epost til påmeldte for  «{{ event.title }}»</h2>
                 </div>
             </div>
         </div>
-        <div class="row">
+        <div class="row" style="font-size: 18px">
```

Comment:
I'd rather see this done through CSS than inline styling.

Reply:
I was wondering how i should do this since i couldnt find a the index.js/css file to the html. Should i create a new bundle in assets/events with a index.js which imports the css document or done some other way?

Reply:
put the css inside of the assets/events/details/ less document since i believe making a new bundle just for these 3 lines so i included it inside an existing css document

Reply:
thought it was fine since since they used inline in. But it makes sense that we shouldnt use inline at all i guess
```html
<div class="row" style="margin-top:30px">
```